src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs

```diff
@@ -546,5 +350,17 @@ private static IEnumerable<TimeSeriesEntry> ChopRelevantRange(TimeSeriesRangeRes
                 yield return value;
             }
         }
+
+        Task<IEnumerable<TValues>> IAsyncSessionDocumentTypedTimeSeries<TValues>.GetAsync(DateTime? @from, DateTime? to, int start, int pageSize, CancellationToken token)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Append(TValues entry)
```

Comment:
Implement that explicitly `IAsyncSessionDocumentTypedTimeSeries<TValues>.Append(TValues entry)`

Reply:
deon